src/utils-matrix.jl

```diff
@@ -321,13 +317,13 @@ function b3_f(
            ) +
```

Comment:
7/14 12:20pm Chris said this sign should be `-`
```suggestion
           ) -
```